app/model.py

```diff
@@ -649,9 +653,13 @@ def put_new(self):
         a new note is created. Also, logs user actions is updated. We should
         never call this method against an existing record."""
         db.put(self)
-        UsageCounter.increment_counter(self.repo, 'note')
         UserActionLog.put_new('add', self, copy_properties=False)
-
+        for note_status in NOTE_STATUS_TEXT:
+            if self.status == note_status:
+                if note_status == '':
```

Comment:
This would be simpler:
```python
if not note_status:
    note_status = 'unspecified'
UsageCounter.increment_counter(self.repo, ['note', note_status])
```